packages/gestalt/src/Masonry.js

```diff
@@ -541,7 +545,7 @@ export default class Masonry<T: { ... }> extends ReactComponent<Props<T>, State<
         ? Math.max(...positions.map((pos) => pos.top + pos.height))
         : 0;
       gridBody = (
-        <div style={{ width: '100%' }} ref={this.setGridWrapperRef}>
+        <div style={{ width: '100%' }} ref={this.setGridWrapperRef} role="list">
           <div className={styles.Masonry} style={{ height, width }}>
```

Comment:
I believe the role="list" needs to be at this level. With the screenreader, I'm hearing "List 1 item" because it's only seeing this div, whereas when I move it, I get "list 15 items"

Reply:
you are absolutely correct! 🤦